inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/iceberg/src/main/java/org/apache/inlong/sort/iceberg/sink/multiple/SchemaChangeUtils.java

```diff
@@ -42,7 +44,7 @@ public class SchemaChangeUtils {
 
     /**
      * Compare two schemas and get the schema changes that happened in them.
-     * TODO: currently only support add column
+     * TODO: currently only support add column,delete column and update column(column type change)
```

Comment:
`update column` include too many situation. Maybe differentiation of fine grains is better, not use `update column`

Reply:
Thanks for the correction